### ISSUE TITLE
Unable to provision dedicated host service

### ISSUE THREAD

**Issue by vijaylakshmi-b** [IBM-Cloud/terraform-provider-ibm#573]
### Terraform Version
terraform -v
v0.11.11
IBM Cloud Provider version 0.16.1

### Affected Resource(s)
ibm_compute_dedicated_host

### Terraform Configuration Files
```
resource "ibm_compute_dedicated_host" "testdedicatedhost" {
	hostname = "test-ded-host"
	domain = "test.com"
	datacenter = "fra02"
	router_hostname = "bcr02a.fra02"
	hourly_billing = "true"
}
```
```
resource "ibm_compute_dedicated_host" "testdedicatedhost" {
	hostname = "test-ded-host"
	domain = "test.com"
	datacenter = "sng01"
	router_hostname = "bcr02a.sng01"
	hourly_billing = "true"
	
}
```
### Debug Output
_Datacenter SNG01:_
https://gist.github.com/vijaylakshmi-b/839ce6070c904e30221e37eb01c57cbc
_Datacenter FRA02:_
https://gist.github.com/vijaylakshmi-b/ac96cf1d0dedf087115fc5e5bea36a50

### Expected Behavior
Dedicated host provisioning should be successful

### Actual Behavior
Datacenter **sng01**(Singapore) has two system configurations supported on SoftLayer:

- 56 Cores X 242 RAM X 1.2 TB

- 56 Cores X 484 RAM X 1.2 TB
However, terraform does not provide an option to select the configuration for Singapore datacenter, and results in the following error:
```
Error during creation of dedicated host: SoftLayer_Exception_Public: Could not place order. An uncaught exception was encountered. Please contact Sales. (HTTP 500)
```
=====================================================================
Datacenter **fra02** (Frankfurt02) only supports _56 Cores X 242 RAM X 1.2 TB_ on the SoftLayer Portal.
However, while creating request body for verify_order API call, the prices fetched are incorrect, resulting in the below error:
```
SoftLayer_Exception_Order_Item_Rule: 56 Cores X 484 RAM X 1.2 TB cannot be ordered with location Frankfurt 2. (HTTP 500)
```
The code is fetching price IDs for both the items, even though there is only one configuration available/supported for that datacenter (which again is not available for the user to view/select).
```
priceItems := []datatypes.Product_Item_Price{}
	for _, item := range productItems {
		for _, price := range item.Prices {
			if price.LocationGroupId == nil {
				priceItem := datatypes.Product_Item_Price{
					Id: price.Id,
				}
				priceItems = append(priceItems, priceItem)
				break
			}
		}
	}
```
The json response of the SoftLayer priceItems API is available in the following Gist:
https://gist.github.com/vijaylakshmi-b/677cd8a51d9c1a6dae99108a3339f679

Both these configurations work fine from the SoftLayer portal.

### Steps to Reproduce
1. `terraform apply`

**Comment by vijaylakshmi-b** [comment]
@Praveengostu @hkantare Request you to look into this

**Comment by hkantare** [comment]
@vijaylakshmi-b When we developed this reosurce there was only one flavor configuration available(56 Cores X 242 RAM X 1.2 TB ) and its was default..We need enhance our provider to take this new changes to support a flavor attribute. We will work on it..

**Comment by hkantare** [comment]
Fix is available as part of master branch https://github.com/IBM-Cloud/terraform-provider-ibm

**Comment by hkantare** [comment]
Fix is available as part of v0.17.2 release
